### ISSUE TITLE
[parallel_shader_compile] Number of threads should not be specified by the Web author

### ISSUE THREAD

**Issue by litherum** [KhronosGroup/WebGL#2772]
https://www.khronos.org/registry/webgl/extensions/KHR_parallel_shader_compile/

```idl
void maxShaderCompilerThreadsKHR(GLuint count);
```
This function should be removed from the extension.

- I can’t imagine any application that would choose to limit itself; Applications often don’t like opting in to slower performance
- There is no way for a Web application to know what a good value to supply is
- Implementations often handle this kind of resource allocation and scheduling deep within the system, depending on things like system load, etc.
- Existing implementations so far seem to ignore this value.

**Comment by litherum** [comment]
Similarly, this call in the example code in the extension should be removed:

```JavaScript
gl.getParameter(ext.MAX_SHADER_COMPILER_THREADS_KHR);
```

**Comment by kenrussell** [comment]
Fixed by #2787 .
